test/index-test.js

```diff
@@ -87,9 +102,8 @@ describe('domEvent', () => {
     setTimeout(() => t.emit(1), 0)
 
     const expected = new Error()
-    return s.tap(_ => { throw expected }).drain()
-      .then(x => assert(false, `expected error, but got event ${x}`),
-        e => assert.strictEqual(expected, e))
+    return observe(_ => { throw expected }, s)
+      .then(x => assert(false, `expected error, but got event ${x}`), eq(expected))
```

Comment:
You can use `fail` here instead of `assert(false, ...)`:

```js
.then(x => fail(`expected error, but got event ${x}`), eq(expected))
```

Reply:
👌